examples/callback.js

```diff
@@ -0,0 +1,38 @@
+// This example uses promises instead of events like "goal_reached" for a cleaner look
+
+const mineflayer = require('mineflayer')
+const pathfinder = require('mineflayer-pathfinder').pathfinder
+const Movements = require('mineflayer-pathfinder').Movements
+const { GoalNear } = require('mineflayer-pathfinder').goals
+const bot = mineflayer.createBot({ username: 'Player' })
+
+bot.once('spawn', () => {
+  // It's better to load plugins after spawning
+  bot.loadPlugin(pathfinder)
```

Comment:
Why ? There should be no issues to use bot.loadPlugin at top level (its designed for it).

Reply:
Hm, alright, I'll change it